lib/improver/tests/nowcasting/optical_flow/test_OpticalFlow.py

```diff
@@ -651,16 +645,30 @@ def test_basic(self):
 
     def test_values(self):
         """Test velocity values are as expected (in m/s)"""
-        ucube, vcube = self.plugin.process(self.cube1, self.cube2)
-        self.assertAlmostEqual(np.mean(ucube.data), -2.12078369915)
-        self.assertAlmostEqual(np.mean(vcube.data), 2.12078369915)
+        ucube, vcube = self.plugin.process(self.cube1, self.cube2, boxsize=3)
+        self.assertAlmostEqual(np.mean(ucube.data), -2.171908358)
+        self.assertAlmostEqual(np.mean(vcube.data), 2.171908358)
+
+    def test_update_smoothing_radius(self):
```

Comment:
It feels like this test is trying to test the change the `self.data_smoothing_radius_km` i.e.
```python
        if cube_time_diff.total_seconds() != 900:
            self.data_smoothing_radius_km *= \
                cube_time_diff.total_seconds()/900.
```
whilst also surreptitiously testing that this error is raised:
```
        if self.boxsize < self.data_smoothing_radius:
            msg = ("Box size {} too small (should not be less than data "
                   "smoothing radius {})")
            raise ValueError(
                msg.format(self.boxsize, self.data_smoothing_radius))
```
Would it be possible to split this testing into two unit tests, so that it's a bit clearer what's being tested?

Reply:
See comment above.  I only want to test the former - I had a choice of errors or warnings to raise...